examples/light-switch-app/efr32/src/binding-handler.cpp

```diff
@@ -202,6 +205,55 @@ CHIP_ERROR ToggleSwitchCommandHandler(int argc, char ** argv)
     return CHIP_NO_ERROR;
 }
 
+/********************************************************
+ * bind switch shell functions
+ *********************************************************/
+
+CHIP_ERROR BindingHelpHandler(int argc, char ** argv)
+{
+    sShellSwitchBindingSubCommands.ForEachCommand(Shell::PrintCommandHelp, nullptr);
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR BindingSwitchCommandHandler(int argc, char ** argv)
+{
+    if (argc == 0)
+    {
+        return BindingHelpHandler(argc, argv);
+    }
+
+    return sShellSwitchBindingSubCommands.ExecCommand(argc, argv);
+}
+
+CHIP_ERROR BindingGroupBindCommandHandler(int argc, char ** argv)
+{
+    VerifyOrReturnError(argc == 2, CHIP_ERROR_INVALID_ARGUMENT);
+
+    EmberBindingTableEntry * entry = Platform::New<EmberBindingTableEntry>();
+    entry->type                    = EMBER_MULTICAST_BINDING;
+    entry->fabricIndex             = atoi(argv[0]);
+    entry->groupId                 = atoi(argv[1]);
```

Comment:
This needs to set `entry->local`, no?  And `entry->clusterId`, unless it's trying to bind all clusters.

Reply:
yes it should